project_double_alias/models/project_project.py

```diff
@@ -23,11 +24,14 @@ def _create_alias_issue(self):
         alias_obj = self.env['mail.alias'].with_context(
             alias_model_name='project.issue',
             alias_parent_model_name=self._name)
+        alias_defaults = {'project_id': self.id}
+        if self.env['project.issue']._fields.get('analytic_account_id'):
+            alias_defaults['analytic_account_id'] = self.analytic_account_id
```

Comment:
Add `# pragma: no cover` at the end of the line (you will need to split the line in 2 for PEP8) to avoid this being taken into account in coveralls (as you can't test this easily).